src/Categories/Category/Instance/SimplicialSet/Properties.agda

```diff
@@ -0,0 +1,212 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Function using (_$_)
+
+module Categories.Category.Instance.SimplicialSet.Properties (o ℓ : Level) where
+
+open import Data.Empty.Polymorphic
+open import Data.Nat using (ℕ)
+open import Data.Fin using (Fin; _<_; fromℕ)
+open import Data.Fin.Patterns
+open import Data.Product using (proj₁)
+
+import Relation.Binary.PropositionalEquality as Eq
+
+open import Categories.Category
```

Comment:
If you could try to be more precise on the imports, especially when you need few things?